core/consensus/babe/babe_util.hpp

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CONSENSUS_BABEUTIL
+#define KAGOME_CONSENSUS_BABEUTIL
+
+#include "consensus/babe/common.hpp"
+#include "consensus/babe/epoch_storage.hpp"
+#include "consensus/babe/types/slots_strategy.hpp"
+#include "primitives/babe_configuration.hpp"
+
+namespace kagome::consensus {
+
+  class BabeUtil final {
```

Comment:
Suggestion to rename it to something narrower, like to something that mentions slots or epochs instead of just utils. Probably will be able to come up with am example in the morning.

Reply:
I wanna refactor it. At least to separate interface and implementation.